pkg/apis/autoscaling/v1/override.go

```diff
@@ -36,10 +37,24 @@ func (in *PodResourceOverrideSpec) Validate() error {
 		return errors.New("invalid value for CPURequestToLimitPercent, must be [0...100]")
 	}
 
+	if in.CPURequestPercent < 0 || in.CPURequestPercent > 100 {
+		return errors.New("invalid value for CPURequestPercent, must be [0...100]")
+	}
+
+	if in.CPURequestPercent != 0 && in.CPURequestToLimitPercent != 0 {
+		return errors.New("you can not specify both CPURequestPercent and CPURequestToLimitPercent")
+	}
+
 	if in.LimitCPUToMemoryPercent < 0 {
 		return errors.New("invalid value for LimitCPUToMemoryPercent, must be a positive value")
 	}
 
+	for _, pattern := range in.TargetedNamespaces {
```

Comment:
I don't think we need this new field, right? There is already a label-based way to activate the web hook in different namespaces. So our deployment would just need to add the labels in all the relevant places.

Reply:
probably not, IIUC the idea that we will have an additional deployment, that will:
- label relevant namespaces
- restart relevant pods to apply scaled CPU request

Reply:
Yes, that sounds like it matches what is proposed in https://github.com/openshift/enhancements/pull/628